libs/base/System/File/ReadWrite.idr

```diff
@@ -54,10 +54,10 @@ fSeekLine (FHandle f)
 
 ||| Get a garbage collected String from a Ptr String and and free the original
 export
-getStringAndFree : HasIO io => Ptr String -> io (Either FileError String)
-getStringAndFree res
+getStringAndFree : HasIO io => File -> Ptr String -> io (Either FileError String)
```

Comment:
I'm not really sure why this is `export`ed from the `File.ReadWrite` module, but since it is, can you add a note about this breaking change to the CHANGELOG?

Reply:
Right. I also have no idea why this is exported (at least, it is not used anywhere in the standard libs), but I did add this change into the changelog. I also realised that we have a separate `System` subsection in the `Base` section of the changelog, and I moved couple already exitstent items there.